pkg/controllers/cluster/import.go

```diff
@@ -213,7 +230,13 @@ func (i *importHandler) importCluster(cluster *fleet.Cluster, status fleet.Clust
 	}
 
 	output := &bytes.Buffer{}
-	err = agentmanifest.AgentManifest(i.ctx, i.systemNamespace, i.systemNamespace, &client.Getter{Namespace: cluster.Namespace}, output, token.Name, &agentmanifest.Options{
+	agentNamespace := i.systemNamespace
```

Comment:
Is this the whole idea that for local cluster rancher will set spec.agentNamespace to `cattle-fleet-local-system` so that fleet-agent will be installed from there and other fleet-agent goes `cattle-fleet-system`?

Reply:
yep.  Two gold stars for you :)